recipe/meta.yaml

```diff
@@ -22,16 +22,18 @@ requirements:
     - pip
     - hatch-jupyter-builder >=0.5.3
     - hatchling >=1.8.1
+    - jupyterlab >=4.0
```

Comment:
This shouldn't be needed because of https://github.com/voila-dashboards/voila/blob/%40voila-dashboards/voila%400.5.7/pyproject.toml#L125 (jupyter npm build doesn't happend because results are already in pypi sdist)